src/components/Card.js

```diff
@@ -27,7 +27,7 @@ export default class Card {
 
     this.cardIsLiked = this.cardIsLiked.bind(this);
   }
-  /* 
+
   handleCardTrashButton() {
     const cardTrashButton = this._element.querySelector(".card__trash-button");
 
```

Comment:
`const verifyDeleteModal = new Popup(".card-delete-verify");`


* `Popup` is not needed in `Card.js` because you can't create instances from it. It's only for **extending**

Every popup should be created only one time and `setEventListeners` should be called also once, otherwise you add listeners again and again to the same elements. This can cause a **`memory leak`**